modules/core/arrow-core-data/src/main/kotlin/arrow/core/MapK.kt

```diff
@@ -91,4 +84,6 @@ fun <K, A, B> Map<K, A>.foldLeft(b: Map<K, B>, f: (Map<K, B>, Map.Entry<K, A>) -
 fun <K, A, B> Map<K, A>.foldRight(b: Map<K, B>, f: (Map.Entry<K, A>, Map<K, B>) -> Map<K, B>): Map<K, B> =
   this.entries.reversed().k().foldLeft(b) { x, y: Map.Entry<K, A> -> f(y, x) }
 
-fun <K, V> mapOf(vararg tuple: Tuple2<K, V>): MapK<K, V> = if (tuple.isNotEmpty()) tuple.map { it.a to it.b }.toMap().k() else emptyMap<K, V>().k()
+// TODO; check duplicate in TupleN
```

Comment:
FIXME!

Reply:
The problem is that there are 2 identical functions. Both of them lived in arrow-core-data and arrow-extras-data. 
I commented out one. We need to decide which should live.
<img width="1417" alt="Screenshot 2019-06-15 at 01 44 30" src="https://user-images.githubusercontent.com/46971368/59544024-259f7780-8f0f-11e9-88b3-1fd65f5fee2c.png">

Reply:
This one lives I'd say. Where's the arrow-extras-data one?

Reply:
Here https://github.com/arrow-kt/arrow/blob/d1b7865c7383c58cee04447a7487190635505c9a/modules/core/arrow-core-data/src/main/kotlin/arrow/core/TupleN.kt#L152
But now MapK is in arrow-core-data, hence the duplication